doc/nrf/ug_matter_device_certification.rst

```diff
@@ -7,12 +7,225 @@ Matter certification
+CSA will ensure that the following certification actions are completed when the certification is granted:
+
+* Certificate of compliance is sent to the certification requester.
+* The newly certified Matter component is listed on the `CSA Certified Products Database`_ page.
+* Matter certified logos are available from the Certification Web Tool dashboard.
+* Certification Declaration file is available for download.
+* Certification record is entered into :ref:`ug_matter_device_dcl`.
+
+Matter certification process variants
+*************************************
+
+The standard Matter certification process has several variants that you can use if you want to certify several products of the same family or you have already obtained the certification.
+
+Product Family certification
+============================
+
+The Product Family certification lets you certify several variants of the same product.
+In this certification path, the first product must be tested according to the original certification process, be it at an ATL or an SVE.
+Then, you can apply for certification of the first product and a number of other products from the same family.
+
+To be considered of the same family, the other products must meet the following criteria:
+
+* All products must share the same device type as the first product.
+* All products must be variants of the first product, which should also be the most feature complete.
+* All products must conform with the Matter specification, regardless of differences.
+
+.. figure:: images/matter_device_certification_process_pf.svg
+   :alt: Matter's Product Family certification overview
+
+   Matter's Product Family certification overview
+
+Certification by Similarity
+===========================
+
+Certification by Similarity lets you certify products that derive from a previously certified product.
+This variant is meant for Matter components that use Matter software that was already certified as part of either the standard procedure or the `Product Family certification`_.
+
+.. figure:: images/matter_device_certification_process_cbs.svg
+   :alt: Matter certification by similarity overview
+
+   Matter certification by similarity overview
+
+Rapid Recertification program
+=============================
+
+.. note::
+      The Rapid Recertification program is currently experimental.
+      You can read more about it in a dedicated document in the `Matter Resource Kit`_.
+
+The Rapid Recertification program lets you perform the product testing using the services of a qualified CSA Member, even your own organization, instead of an ATL.
+This allows for a potentially more streamlined recertification process.
+
+.. figure:: images/matter_device_certification_process_rr.svg
+   :alt: Matter's Rapid Recertification program overview
+
+   Matter's Rapid Recertification program overview
+
 
 .. _ug_matter_device_certification_reqs:
 
-Certification requirements
-**************************
+Dependant certification requirements
```

Comment:
I am thinking, maybe this section should be moved to the application requirement to have it under the same section as Matter Security Attestation

Reply:
I'd prefer avoiding moving the whole section there. Once we add LTE or any other new dependent certification subsection, we'll end up with very long subsection contents. There is already a short mention of these requirements there...